app/services/capital_collator_and_assessor.rb

```diff
@@ -2,19 +2,18 @@ class CapitalCollatorAndAssessor
   class << self
     def call(assessment)
       data = collate_applicant_capital(assessment)
-      assessment.capital_summary.update!(data.except(:total_vehicle))
       if assessment.partner.present?
         partner_data = collate_partner_capital(assessment)
```

Comment:
Think it would be nice if this function returned a PersonCapitalSubtotals object, rather than having to convert from a hash at this layer

Reply:
Yes! As soon as you say it, it's obvious it should be like that.